extensions/pi-hole/package.json

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
There should be JSON schema reference at the head of the file. Please don't remove it.